spec/lib/vmdb/global_methods_spec.rb

```diff
@@ -85,6 +85,12 @@ class TestClass
 
       it "with a timezone" do
         user = FactoryGirl.create(:user, :settings => {:display => {:timezone => "Pacific Time (US & Canada)"}})
+        subject.get_timezone_for_userid(user).should == "Pacific Time (US & Canada)"
+      end
+
+      # currently only used in 1 place
```

Comment:
Is [this](https://github.com/ManageIQ/manageiq/blob/5f47999499ec2bc5e3b1d3eb3785964d7ecdbb81/app/controllers/application_controller.rb#L913-L924) the one place?

Looks like it can easily make better use of this new take on the method, even now:

```ruby
def set_user_time_zone
  @tz = get_timezone_for_userid(current_user || params[:user_name].presence)
  session[:user_tz] = Time.zone = @tz
end
```

... which is actually a short journey from:

```ruby
def set_user_time_zone
  user = current_user || (params[:user_name].presence && User.find_by_userid(params[:user_name]))
  session[:user_tz] = Time.zone = @tz = get_timezone_for_user(user)
end
```

(is `@tz` actually used anywhere?)